eta-meeting-organizer-frontend/src/app/models/user.model.ts

```diff
@@ -10,7 +10,7 @@ export interface User {
 }
 
 export enum Role {
-  ADMIN,
-  USER,
-  READER
+  ADMIN = 'ADMIN',
+  USER = 'USER',
```

Comment:
Biztos szukseges?

Reply:
Typescriptben találtam ezt a megoldást, hogy lehet még az enumhoz hozzárendelni értéket. Enélkül amikor a header komponensben csináltam az if-et (this.user.role === Role.ADMIN) külön a Role-t toString()-gel még alakítani kellett, mert mindig false-t dobott az ADMIN-ra is.